src/server/debugging.ts

```diff
@@ -506,11 +520,14 @@ export async function getContainerInfo(
     for (const process of kernelData.processes) {
         const gdbProcess = gdbProcesses.find((proc) => proc.pid === process.pid);
         if (!gdbProcess) {
-            console.warn(
-                `[container pid ${containerPid}] Could not find gdb inferior with pid ${process.pid}`,
-                'Kernel data:', kernelData,
-                'Gdb processes:', gdbProcesses,
-            );
+            // we do not expect to find a GDB process for zombie processes, so don't warn
```

Comment:
Hmm. Do you think it would make sense to warn if we *do* find a GDB process for zombie processes? I'm trying to think if there are any valid situations where GDB would see a zombie process, and I can't think of any yet.

Reply:
I do think a GDB process for a zombie process is a bit weird, as nothing would be executing. I updated the logging code to expect (and log if it does not find that) a GDB process will exist if and only if the client process is non-zombified.